client/src/Widgets/ActivityTable.tsx

```diff
@@ -82,195 +285,16 @@ export default forwardRef(function ActivityTable(
-                            (e) => {
-                              e.preventDefault();
-                              e.stopPropagation();
-                            },
-                            { capture: true, once: true },
-                          );
-                        }}
-                      />
-                    </Editable>
-                    {activity.isPublic || activity.isShared ? (
-                      <Icon
-                        as={BsPeopleFill}
-                        color="#666699"
-                        boxSize={5}
-                        verticalAlign="baseline"
-                      />
-                    ) : null}
-                  </HStack>
-                  <Show below="md">
-                    {showAssignmentStatus ? (
-                      <Text>{assignmentStatusString}</Text>
-                    ) : null}
-                  </Show>
-                </Td>
-                {showPublicStatus ? (
-                  <Td>{activity.isPublic ? "Public" : "Private"}</Td>
-                ) : null}
-                <Show above="md">
-                  {showAssignmentStatus ? (
-                    <Td>{assignmentStatusString}</Td>
-                  ) : null}
-                </Show>
-                {(!suppressAvatar && !activity.authorRow) ||
-                (showOwnerName && !activity.authorRow) ? (
-                  <Td>
-                    <HStack>
-                      {suppressAvatar || activity.authorRow ? null : (
-                        <Tooltip label={activity.ownerName}>
-                          <Avatar size="sm" name={activity.ownerName} />
-                        </Tooltip>
-                      )}
-                      {showOwnerName && !activity.authorRow ? (
-                        <Text>{activity.ownerName}</Text>
-                      ) : null}
-                    </HStack>
-                  </Td>
-                ) : null}
-                <Td p="0" m="0" textAlign="right">
-                  {activity.menuItems ? (
-                    <Menu>
-                      <MenuButton
-                        data-test="Card Menu Button"
-                        _focus={{ boxShadow: "outline" }}
-                        ref={ref}
-                        padding="10px"
-                        onClick={(e) => e.stopPropagation()}
-                      >
-                        <Icon color="black" as={GoKebabVertical} boxSize={6} />
-                      </MenuButton>
-                      <MenuList
-                        zIndex="1000"
-                        onClick={(e) => e.stopPropagation()}
-                      >
-                        {activity.menuItems}
-                      </MenuList>
-                    </Menu>
-                  ) : null}
-                </Td>
-              </Tr>
-            );
-          })}
+          {content.map((activity) => (
+            <ActivityRow
+              activity={activity}
```

Comment:
We need a unique `key` here. Using `activity.id` will now work (no need to treat folder differently, now that we have UUIDs for the ids).